bokehjs/src/coffee/models/tools/gestures/pan_tool.coffee

```diff
@@ -39,44 +39,44 @@ class PanToolView extends GestureTool.View
     new_dx = dx - @last_dx
     new_dy = dy - @last_dy
 
-    hr = _.clone(frame.get('h_range'))
```

Comment:
Apparently there is Backbone.Model's method `clone()` which does exactly what should be done, so `obj.clone()` does the job.